base/docs/helpdb/Base.jl

```diff
@@ -130,13 +130,28 @@ Obtain the contents of an `IOBuffer` as an array, without copying. Afterwards, t
 takebuf_array
 
 """
-    download(url,[localfile])
+    download(url [localfile] [silent=false])
```

Comment:
Why did you remove the comma? OTC, you need to add one before `silent`. Also, since there is `=false`, the brackets are not needed.

Same below.